.circleci/config.yml

```diff
@@ -124,6 +124,25 @@ jobs:
       - run: |
           kurtosis run ./simple-api/kurtosis-package/
 
+  build_deploy_simple_api:
+    executor: ubuntu_vm
+    steps:
+      - checkout
+      # If this sub-repo hasn't changed, don't run the tests
+      - run: |
+          if git --no-pager diff --exit-code origin/main...HEAD -- ':.circleci' ':simple-api'; then
+            circleci-agent step halt
+          fi
+      - run: echo "${DOCKER_PASSWORD}" | docker login -u ${DOCKER_USERNAME} --password-stdin
+      - run: simple-api/scripts/build.sh
+      - run: |
+          set -euo pipefail
+          version_to_publish="latest"
+          image_org_and_repo="kurtosistech/kurtosis-simple-api"
+          image_name_with_version="${image_org_and_repo}:${version_to_publish}"
+          docker tag "${image_name_with_version}" "${image_name_with_version}"
```

Comment:
Why do we have this line? I believe this isn't doing anything. You're basically running something like:
```
docker tag kurtosistech/kurtosis-simple-api:latest kurtosistech/kurtosis-simple-api:latest
```
If the image is already named `kurtosistech/kurtosis-simple-api:latest` I guess there's no need to tag it with that same name :)

Reply:
(in the monorepo we're doing `docker tag "${image_name_with_version}" "${image_name_no_version}"`) because we want the image we're publishing to have 2 tags, one `:latest` and one with the actual version, like `:0.67.2` whatever)